src/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -64,6 +64,7 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
         this.deviceType = DEVICETYPE_WEBGPU;
         this.precision = 'hiphp';
         this.maxSamples = 4;
+        this.maxTextures = 16;
```

Comment:
Is this just for backwards compatibility or is it also a limit enforced by WebGPU?

Reply:
webgpu version 1 does not expose limits yet (in the next release it will) .. so these are internally set limits to match webgl2 for now. All subject to change as we see fit.